src/functions/template-tags/organizer.php

```diff
@@ -332,24 +362,53 @@ function tribe_get_organizer_website_url( $postId = null ) {
 	function tribe_get_organizer_website_link( $post_id = null, $label = null ) {
 		$post_id = tribe_get_organizer_id( $post_id );
 		$url     = tribe_get_event_meta( $post_id, '_OrganizerWebsite', true );
+
+		/**
+		 * Filter the organizer link target attribute.
+		 *
+		 * @since TBD
+		 *
+		 * @param string the target attribute string. Defaults to "_self".
+		 */
+		$target = apply_filters( 'tribe_get_event_organizer_link_target', '_self' );
+		$rel    = ( '_blank' === $target ) ? 'noopener noreferrer' : 'external';
+
+		/**
+		 * Filter the organizer link label
+		 *
+		 * @since TBD
+		 *
+		 * @param string the link label/text.
+		 */
+		$label  = apply_filters( 'tribe_get_organizer_website_link_label', $label );
```

Comment:
:warning: Equals sign not aligned correctly; expected 1 space but found 2 spaces

<sub>hash: a2d01cb6de424d1d146a87c9ab986a3d</sub>